Tasks/VsTest/vstest.ts

```diff
@@ -485,6 +490,7 @@ function cleanFiles(responseFile: string, listFile: string, testCaseFilterFile:
 
 function deleteVstestDiagFile(): void {
```

Comment:
can we rename this to say UploadAndDelete

Reply:
Yes